tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -193,95 +193,95 @@ Synchronization features like etag propagation, setting mtime and locking files
 - [apiWebdavLocks/requestsWithToken.feature:156](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature#L156)
 - [apiWebdavLocks/requestsWithToken.feature:157](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature#L157)
 - [apiWebdavLocks/requestsWithToken.feature:162](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature#L162)
+- [apiWebdavLocks2/resharedSharesToShares.feature:31](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L31)
+- [apiWebdavLocks2/resharedSharesToShares.feature:32](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L32)
```

Comment:
are those tests failing now that were not failing before?

Reply:
Just line number changes from test scenario refactoring in https://github.com/owncloud/core/pull/39919
Lines 33 to 36 became 31 to 34.